ddsrecorder/src/cpp/main.cpp

```diff
@@ -277,6 +267,39 @@ int main(
         // Load configuration from YAML
         eprosima::ddsrecorder::yaml::RecorderConfiguration configuration(file_path);
 
+        if (log_verbosity.is_set() && !configuration.log_configuration.verbosity.is_set())
+        {
+            configuration.log_configuration.verbosity.set_value(log_verbosity);
+            std::cout << "Verbosity: " << configuration.log_configuration.verbosity << std::endl;
+        }
+
+        if (log_filter.is_set() && !configuration.log_configuration.filter.is_set())
+        {
+            configuration.log_configuration.filter = log_filter;
+        }
+
+        if (!log_filter.is_set() && !configuration.log_configuration.filter.is_set())
+        {
+            configuration.log_configuration.filter.set_value({
+                {eprosima::utils::VerbosityKind::Error, ""},
+                {eprosima::utils::VerbosityKind::Warning, "(DDSRECORDER|DDSPIPE)"},
+                {eprosima::utils::VerbosityKind::Info, ""}});
```

Comment:
I believe the `VerbosityKind::Info` filter should be set to 'DDSRECORDER'. Suppose a user set the verbosity to Info but didn't set the filter. Printing all the logInfos while filtering the logWarnings feels a bit weird.